README.md

```diff
@@ -2,3 +2,18 @@
 
 A shard is what we call a crystal libraries, and this is the website that lists
 them all.
+
+# deploying
+
+To experiment with this, you can post this to your own heroku site.
+
+You can read about [deploying to crystal] or just follow these steps:
```

Comment:
We're not really "deploying *to* crystal", right? What about "deploying yo heroku", or "deploying a Crystal app"?